packages/snack-sdk/src/transports/types.ts

```diff
@@ -45,9 +45,7 @@ export type SnackTransportEventConnect = {
 export type SnackTransportEventDisconnect = {
   type: 'disconnect';
   connectionId: string;
-  data: {
-    timedout: boolean;
-  };
+  data: any;
```

Comment:
Do we need this `data` property on the disconnect event? If not, I would omit it. The type doesn't really add much value if we don't use it. (and if we do use it, might be worth adding proper types)

Reply:
this is required because `SnackTransportEventDisconnect` is one kind of `SnackTransportEvent`. we have code like
```
const { data } = (event as SnackTransportEvent);
```
so we need the data key here. leaving a comment anyway: https://github.com/expo/snack/pull/376/commits/d515e4dc068c9f33aaab12fc169b272b81a504c9#diff-515958a3d7297a4dd129e86e7a4f99dc5a4d2071958d80bb6479be90ce27c0aaR48